pandas/tests/reductions/test_reductions.py

```diff
@@ -679,6 +679,23 @@ def test_empty_multi(self, method, unit):
         expected = Series([1, np.nan], index=["a", "b"])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("method", ["mean"])
+    @pytest.mark.parametrize("dtype", ["Float64", "Int64", "boolean"])
+    def test_ops_consistency_on_empty_nullable(self, method, dtype):
```

Comment:
an you add min_count as a paramterizatoon (result i think is the same), also NA

Reply:
> add min_count as a paramterizatoon

As mentioned above, min_count is not a valid keyword for mean()

> , also NA

Can you clarify this? Parametrize for different NA values? Note that this test is for nullable dtypes, so there is only a single possible NA value (not for construction of course, but we are not testing construction here)